### PR TITLE
Revise upload/preview/edit

### PR DESCRIPTION
**Pull request by kiroj305**
Implemented recent design changes, and fixed some more issues.

@kjawadDeveloper2 , I've checked ui on tablet emulator. You can check yourself if you want. Thanks

### CODE REVIEW

File: lib/utils/constants.dart [hunk 1]
```diff
@@ -122,41 +70,118 @@ const kMaxRoyalty = 99.99;
...
-const kJunoSymbol = "ujunox";
-const kLunaSymbol = "uluna";
-const kUSTSymbol = "uusd";
+const kPylonSymbol = 'upylon';
+const kUsdSymbol = 'ustripeusd';
+const kAtomSymbol = 'uatom';
+const kEuroSymbol = 'eeur';
+const kAgoricSymbol = 'urun';
+const kJunoSymbol = 'ujunox';
+const kLunaSymbol = 'uluna';
```
Comment (mustafapylons) [inline lib/utils/constants.dart]: We are removing LUNA AND UST.

**Review by mustafapylons — COMMENTED** [review]
One small comment, otherwise code is good.

File: lib/utils/constants.dart (hunk 1 above)
Comment (kiroj305) [reply lib/utils/constants.dart]: @mustafapylons , removed all appearances of luna and ust

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Comment by kjawadDeveloper2** [comment]
Checking it now.

**Comment by kjawadDeveloper2** [comment]
down
The usd and add button looks greater in size than design. It also looks like there is some elevation given to these button as they are not exactly placed completely on the below fields.

**Comment by kjawadDeveloper2** [comment]
The number of editions in editions in design is different than the app.

**Comment by kjawadDeveloper2** [comment]
The hint in each field is not vetrically center aligned.

**Comment by kjawadDeveloper2** [comment]
down2
the tags are scrollable in the app. While in the design its moved to next line.

**Comment by kjawadDeveloper2** [comment]
Maybe we can go in session with QA and confirm the design from them before merging it in the main code?

**Review by afti-githobo — APPROVED** [review]

**Review by kjawadDeveloper2 — APPROVED** [review]
LGTM

**Pull request merged**